lib/active_model/serializer/adapter/json_api.rb

```diff
@@ -19,10 +20,11 @@ def initialize(serializer, options = {})
 
         def serializable_hash(options = nil)
           options ||= {}
+
           if serializer.respond_to?(:each)
-            serializable_hash_for_collection(serializer, options)
```

Comment:
I'd kind of like to simplify defining `def serializable_hash` in subclasses so that we can either `def serializable_hash(*); super; whatever` or `def _serializable_hash(options)` a method called by `serializable_hash`, or alternatively, even just use `read_attribute_for_serialization`